src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -3385,7 +3385,7 @@ paths:
         - jobs
         - repository
       description: >
-        Returns a list of all of the jobs the caller has access to
+        Returns a list of all recently jobs submitted that the caller has access to
```

Comment:
I think this should be "recently submitted jobs" :)

Reply:
Gah, you are correct